meetings/19-February-2025/agenda.md

```diff
@@ -0,0 +1,37 @@
+# Agenda for Safety Critical Rust Consortium Meeting, 19 February 2025, London UK
+
+The second meeting of the Safety Critical Rust Consortium meeting will be held on 19 February 2025 from 9 AM - 4 PM GMT/UTC in London, UK. Below is the proposed agenda. Additional suggestions welcome; please suggest a change or comment on the pull request associated with this agenda item.
+
+- Welcome (@JoelMarcey)
+- Consortium Logistics (@JoelMarcey)
+  - Membership Moving Forward
+  - Public information in the GitHub repo
+    - Names and Companies?
+  - Welcoming new members
+- Subcommittee Reports
+  - Coding Guidelines (@PLeVasseur)
+    - Status
+    - Current Work items
+    - Goals moving forward
+    - Other full consortium discussion items
+  - Tooling (@alexandruradovici)
+    - Status
+    - Current Work items
+    - Goals moving forward
+    - Other full consortium discussion items
+  - Form new subcommittees? (@JoelMarcey, all)
+- Lunch (on your own)
+- Rust Specification Update and Discussion (@JoelMarcey)
+- Breakout groups (1 hour)
+  - Coding Guidelines (@PLeVasseur)
+  - Tooling (@alexandruradovici)
+  - Specification (@JoelMarcey)
+- Close (@JoelMarcey)
+  - Post-mortem
+    - What worked well?
+    - What can be improved?
+  - Next meeting
+    - RustConf?
```

Comment:
What about RustNL?

Reply:
Let's talk about this at the meeting. I was thinking twice yearly at 6-month cadences, and always having one at RustConf. But I am willing to consider more meetings or different cadences.

Reply:
I would be in favor of RustNL as well. I'm not sure I (a transgender woman) can safely travel to the US given the current political situation there.